terraform/alerting/workspace.tf

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "null_resource" "manual-step-to-enable-workspace" {
+  # TODO: remove once
+  # https://github.com/hashicorp/terraform-provider-google/issues/2605 is
+  # fixed.
+  provisioner "local-exec" {
+    command = <<EOF
+    echo -e '\033[0;31m>>>> WARNING WARNING WARNING!'
```

Comment:
Let's not rely on ANSI sequences here in case anyone is running Windows

Reply:
Interesting. Do we need to worry about that? Will `echo` work in Windows?

Reply:
Echo works in powershell but not cmd. But in general, colors and special formatting should be avoided unless strictly required.

Reply:
TIL!